docs/content/en/docs/installation/metrics.md

```diff
@@ -16,15 +16,18 @@ For the full list, refer to [metrics reference]({{< ref "/docs/reference/metrics
 
 ### Kubernetes
 
-In a [Kubernetes installation]({{< ref "/docs/installation/kubernetes" >}}), metrics are enabled by default and exposed
-via `tetragon` service at endpoint `/metrics` on port `2112`.
+In a [Kubernetes installation]({{< ref "/docs/installation/kubernetes" >}}), **metrics are enabled by default** and
+exposed via `tetragon` service at endpoint `/metrics` on port `2112` (Agent) and `2113` (Operator).
```

Comment:
Operator metrics are exposed by `tetragon-operator-metrics` service, probably worth mentioning it if we mention operator port.